README.md

```diff
@@ -248,6 +248,15 @@ Take a look at `BTCardFormViewController.m` to see examples of using the form fi
 
 Start with [**'Hello, Client!'**](https://developers.braintreepayments.com/ios/start/hello-client) for instructions on basic setup and usage.
 
+## Versions
+
+This SDK abides by our Client SDK Deprecation Policy. For more information on the potential statuses of an SDK check our [developer docs](http://developers.braintreepayments.com/guides/client-sdk/deprecation-policy).
```

Comment:
![abides](https://user-images.githubusercontent.com/22104233/104608487-9062e880-5647-11eb-9988-7e98ce9108ab.gif)

Reply:
```suggestion
This SDK abides by our Client SDK Deprecation Policy. For more information on the statuses of an SDK, check our [developer docs](http://developers.braintreepayments.com/guides/client-sdk/deprecation-policy).[Minimum supported versions](https://developers.braintreepayments.com/guides/client-sdk/setup/ios/#requirements) are also available in our developer docs.
```

Reply:
One note about my suggested changes - these link to the minimum supported version of the Core SDK. I'm wondering if we should use the link to Drop-In's [setup page](https://developers.braintreepayments.com/guides/drop-in/setup-and-integration/ios/v5#setup) instead. Only think if the Drop-In setup page doesn't mention minimum supported versions... maybe we should update that.

Reply:
Good call - I added a JIRA (#DTBTSDK-720) to update the Drop-In Developer Docs with the minimum supported versions. 

Once we do - we can make this change to the README. How does that sound, and we add that in a later PR?